Source/Extensions/Blazorise.Charts/wwwroot/blazorise.charts.js

```diff
@@ -1,11 +1,5 @@
 window.blazoriseCharts = {
-    setChartData: (canvasId, type, data, options, dataIsString, optionsIsString) => {
-        if (dataIsString)
```

Comment:
maybe leave json strings for now?

Reply:
Done. Although had to pass string as separate parameter (to typed value on JS.cs caller)